src/commands/coin/coinInfo.ts

```diff
@@ -0,0 +1,46 @@
+import { Message, MessageEmbed } from 'discord.js';
+import { CommandoClient, CommandoMessage } from 'discord.js-commando';
+import { BaseCommand } from '../../utils/commands';
+import { EMBED_COLOUR } from '../../utils/embeds';
+
+class CoinInfo extends BaseCommand {
+  constructor(client: CommandoClient) {
+    super(client, {
+      name: 'coin-info',
+      aliases: ['ci', 'coininfo', 'codeycoin info'],
```

Comment:
`'codeycoin info'`: i don't think this works because it has a space, but it's ok we don't need it :)

Reply:
oh whooops meant to have a dash, i'll change this/get rid of it, thank you